import_export/formats/base_formats.py

```diff
@@ -84,7 +85,13 @@ def get_title(self):
     def create_dataset(self, in_stream, **kwargs):
         return tablib.import_set(in_stream, format=self.get_title())
 
-    def export_data(self, dataset, escape_output=False, **kwargs):
+    def export_data(self, dataset, **kwargs):
+        # remove the deprecated `escape_output` param if present
+        kwargs.pop("escape_output", None)
+        if kwargs.pop("escape_html", None):
+            self._escape_html(dataset)
+        if kwargs.pop("escape_formulae", None):
+            self._escape_formulae(dataset)
```

Comment:
I do agree with you, but the `dataset` object supports changing data in place, so it is done here.  It might be confusing to return it as well.  It is an unusual pattern but I'm personally ok with it in an internal non public function.